pkg/apis/provisioning/v1alpha5/requirements.go

```diff
@@ -220,7 +220,7 @@ func (r Requirements) Validate() (errs error) {
 		if requirement.Operator == v1.NodeSelectorOpDoesNotExist && (r.hasRequirement(withKeyAndOperator(requirement.Key, v1.NodeSelectorOpIn)) ||
 			r.hasRequirement(withKeyAndOperator(requirement.Key, v1.NodeSelectorOpNotIn)) ||
 			r.hasRequirement(withKeyAndOperator(requirement.Key, v1.NodeSelectorOpExists))) {
-			errs = multierr.Append(errs, fmt.Errorf("operator %s and %s conflict for key %s", v1.NodeSelectorOpDoesNotExist, v1.NodeSelectorOpDoesNotExist, requirement.Key))
+			errs = multierr.Append(errs, fmt.Errorf("operator %s cannot coexist with other opeartors for key %s", v1.NodeSelectorOpDoesNotExist, requirement.Key))
```

Comment:
I like the previous error message. What's the goal of this change?

Reply:
This if condition actually catches three cases (i.e., <NotExist, NotIn>, <NotExist, In>, <NotExist, Exists>). So the previous message says `operator DoesNotExist and DoesNotExist conflict for key XXX` because we didn't distinguish which second operator it is conflicting with.

Reply:
How is this possible? The or statement only checks In, NotIn, and Exists, so there double `DoesNotExist` case should never trigger.

Reply:
The fmt.errorf second and third parameters are all v1.NodeSelectorOpDoesNotExist.

Reply:
Oh I see -- and since we're using an OR statement, we don't know which one it is.

We should fix the spelling on opeartor -> operator and then we're good to go :)

Reply:
omg, sorry for the typo. done.